### ISSUE TITLE
Add the requests module 

### ISSUE THREAD

**Issue by iam-py-test** [raxod502/python-in-a-box#3]
It would be nice if the requests module was installed, but I am not sure how easy that is

**Comment by raxod502** [comment]
This seems out of scope, as the point of this project isn't to provide a full Python development environment, it's to demonstrate how straightforward it is to produce a minimal code execution environment in a web application. See, however, [Riju](https://github.com/raxod502/riju), for which this would not be out of scope.

**Comment by raxod502** [comment]
This thread is being closed automatically by [Tidier](https://github.com/raxod502/tidier) because it is labeled with "waiting on response" and has not seen any activity for 90 days. But don't worry—if you have any information that might advance the discussion, leave a comment and I will be happy to reopen the thread :)
